docker/test/fuzzer/run-fuzzer.sh

```diff
@@ -234,6 +234,7 @@ quit
     # SC2046: Quote this to prevent word splitting. Actually I need word splitting.
     # shellcheck disable=SC2012,SC2046
     timeout -s TERM --preserve-status 30m clickhouse-client \
+        --max-memory-usage-for-client=1000000000 \
```

Comment:
Should it be `1073741824` or is the decimal system implied?

Reply:
Both 1 GB and 1 GiB are ok for this purpose.